azure_arc_servers_jumpstart/oci/ubuntu/terraform/main.tf

```diff
@@ -0,0 +1,162 @@
+variable "tenancy_ocid" {}
+variable "user_ocid" {}
+variable "fingerprint" {}
+variable "private_key_path" {}
+variable "compartment_ocid" {}
+variable "region" {}
+variable "ssh_public_key" {}
+
+resource "azurerm_resource_group" "azure_rg" {
+  name     = var.azure_resource_group
+  location = var.azure_location
+  tags = {
+    project = "jumpstart_azure_arc_servers"
+  }
+}
+
+variable "ad_region_mapping" {
+  type = map(string)
+  default = {
+    us-phoenix-1 = 2
+    us-ashburn-1 = 1
+  }
+}
+
+variable "images" {
+  type = map(string)
+
+  default = {
+    # See https://docs.us-phoenix-1.oraclecloud.com/images/
+    # Oracle-provided image "Ubuntu-Linux-18.4-lts"
+    us-phoenix-1   = "ocid1.image.oc3.us-gov-phoenix-1.aaaaaaaaqhxpqz5jrml5twxtoe7z37fw7qnprlpi4gbx6rgprfwg3vpup2zq"
+    us-ashburn-1   = "ocid1.image.oc1.iad.aaaaaaaah2ms24xr6slrtpppgaipfixozl7utwnf2qwqonb2muk4g43wfzgq"
+  }
+}
```

Comment:
Consider moving variable declarations to standalone variables.tf file.

Reply:
@JerryMSFT this is a Terraform best practice. Please implement.